src/Client/PullPayment.php

```diff
@@ -0,0 +1,336 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Client;
+
+class PullPayment extends AbstractClient
+{
+    /*
```

Comment:
This docblock looks a bit strange, which client do you use. As this seems to be a class description it would belong above the class on line 6.

For general code style we use PSR-12. You can run php codestyle check with `composer cs-check` and `composer cs-fix`; It won't fix docblocks though. 

Generally we use docblox syntax from here that is also enforced by e.g. PHPStorm  https://docs.phpdoc.org/3.0/guide/references/phpdoc/basic-syntax.html#basic-syntax

Reply:
Ok thanks I will clean this up. I am using VSCode, but I have never made comments in code before, not quite sure the right way. I will read up on this.